src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject_IVsENCRebuildableProjectCfg.cs

```diff
@@ -18,69 +19,44 @@ public int HasCustomMetadataEmitter(out bool value)
         }
 
         public int StartDebuggingPE()
-        {
-            return _editAndContinueProject.StartDebuggingPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
```

Comment:
>throw ExceptionUtilities.Unreachable; [](start = 15, length = 37)

Does this mean we're removing the EnC support from the legacy projects?

Reply:
No. It's using a different code path now.